### ISSUE TITLE
Obsolete "teams-for-linux" Flatpak for official client?

### ISSUE THREAD

**Issue by hadess** [flathub/com.github.IsmaelMartinez.teams_for_linux#14]
It would be useful to find a way to automatically migrate folks from the unofficial, and now obsolete, client to the official one (even if unofficially packaged).

There's a few examples using the eol-rebase flathub.json key:
https://github.com/search?q=org%3Aflathub+eol-rebase&type=commits

Original bug: https://github.com/IsmaelMartinez/teams-for-linux/issues/392

**Comment by hadess** [comment]
At the very least, the branding needs to make it clear that this is not Microsoft's Teams, with a different icon. In the search results it looks like there's the current stable, and the preview version of Teams:
https://flathub.org/apps/search/teams

**Comment by IsmaelMartinez** [comment]
MS didn't complain us using the icon and we state is not the official client. Hopefully MS does fix the many issues that makes "teams-for-linux" still been in use and give me then some of my life back!

**Comment by hadess** [comment]
> MS didn't complain us using the icon and we state is not the official client.

That isn't something that somebody can guess from either the name or the icon, quite the opposite. I wouldn't have filed this bug if that wasn't the case.

**Comment by IsmaelMartinez** [comment]
Feel free to create a new icon then in https://github.com/IsmaelMartinez/teams-for-linux/tree/develop/build/icons, create a PR and I will merge it. Thanks!

**Comment by hadess** [comment]
I don't think it would be a very good one, and I don't think that I should be the one fixing your use of trademarked/copyright material. I don't know, put a penguin on it or something.

**Comment by IsmaelMartinez** [comment]
you can be part of the community if you want to...

**Comment by hadess** [comment]
@nedrichards @bilelmoussaoui any comments, as some of the original reviewers? https://github.com/flathub/flathub/pull/1020

The official MS app was reviewed 6 months afterwards:
https://github.com/flathub/flathub/pull/1292

**Comment by hadess** [comment]
> you can be part of the community if you want to...

I have no idea what you're talking about.

**Comment by IsmaelMartinez** [comment]
It's ok @hadess, its just that currently the teams-for-linux is in maintenance mode only. 

I will see if/when I got time to change those logos but it will take me probably months(I got no time nor get paid for this). I will have appreciated some help, as many others have done, but also understand people might not have the abilities and/or the time to do it.

I did try to contact multiple times MS regarding their linux support in early days (almost 3 years ago) mentioning the use of the icon and me willing to help getting MS Teams in Linux but got 0 response from them.

I will keep an 👂  on this repository and if the maintainers decide to consider this an EoL and migrate people to the Official MS version, I will remove the reference in the teams-for-linux repository. The opposite applies, if I mark the teams-for-linux as EoL and stop maintaining it, I will notify here and try to help on that migration.
